### DISCUSSION TITLE
What is the reason to delete the branch mickledore

### DISCUSSION THREAD

**Discussion by junzhuimx** [Freescale/meta-freescale#1550] [Q&A] [answered]
@otavio 
The branch mickledore (Yocto 4.2) is deleted from 2-week ago.
- Will cut from master later?
- What is the branch strategy of meta-freesclare? In the past years, we followed with poky and meta-oe to create branches for every Yocto release.

**Comment by otavio** [comment, marked as answer]
@junzhuimx indeed. We reworked the code to allow it to work with the `master` branch. The reasons are the same as https://github.com/Freescale/meta-freescale/discussions/1329; we prefer to keep a single branch that is compatible with it.

We aim to make a branch for the next Yocto Project LTS release, as companies will likely commit their products to LTS releases.
